src/hotspot/share/gc/shared/c2/barrierSetC2.cpp

```diff
@@ -217,7 +217,7 @@ Node* BarrierSetC2::load_at_resolved(C2Access& access, const Type* val_type) con
                             unaligned, mismatched, unsafe, access.barrier_data());
       load = kit->gvn().transform(load);
     } else {
-      load = kit->make_load(control, adr, val_type, access.type(), adr_type, mo,
```

Comment:
No similar change to `BarrierSetC2::store_at_resolved()`?

Reply:
I missed that. Fixed now.